lib/novu/service.ts

```
@@ -51,13 +52,51 @@ interface TriggerResult {
+  workflowId: string,
+  recipients: string | string[],
+  payload: NovuPayload,
+  dedupeKey?: string,
+): string {
+  const canonicalPayload = JSON.stringify(
+    Object.fromEntries(
+      Object.entries(payload).sort(([a], [b]) => a.localeCompare(b)),
+    ),
+  );
+  const recipientKey = Array.isArray(recipients)
+    ? [...recipients].sort().join(",")
```

ESLint

[error] Use recipients.toSorted() instead of copying and sorting

(e18e/prefer-array-to-sorted)